.github/workflows/ci.yml

```
@@ -282,6 +287,13 @@ jobs:
           sudo make install
         working-directory: src/github.com/containerd/containerd
 
+      - name: Tests
+        env:
+          GOPROXY: direct
+        run: |
+          make test
+          sudo -E PATH=$PATH GOPATH=$GOPATH GOPROXY=$GOPROXY make root-test
```

Are the ` GOPATH=$GOPATH GOPROXY=$GOPROXY` needed? I think they should be preserved by `-E` (only `PATH` is a special case)

```suggestion
          sudo -E PATH=$PATH make root-test
```

---

to be honest I was just copying from integration section below. :)

---

oh, good one. Looks like those don't use `sudo -E` (not sure why that was not used), so they would only get the env-vars that are explicitly set 🤔